README.md

```diff
@@ -1,15 +1,314 @@
-# Shutter SDK
-Custom ethersjs Provider for Rolling Shutter on Optimism.
+# Shutter Shop-SDK
 
-## Build/Install
+Custom ethersjs Provider for Shutter on Optimism.
+
+## Introduction
+
+`shop-sdk` is a custom [`ethersjs`](https://docs.ethers.org/v5/api/providers/) provider, that
+allows to send shutter encrypted ("shutterized") transactions on Shutter-Optimism.
+
+Shutter-Optimism is an [Optimism](https://www.optimism.io/) testnet, that provides a mechanism to
+include Shutter encrypted transactions.
+
+[Shutter](http://shutter.network/) is a distributed key generation (DKG) threshold encryption
+protocol, that enables applications and/or users to encrypt arbitrary data, _commit_ the encrypted
+message (i.e. by sending it in a transaction) and have it _revealed_ after a certain condition is
+met (i.e. once the position of the transaction in a future block is confirmed). Shutter's
+threshold decryption scheme, where a threshold of `n` out of `m` key holders ("keypers") need to
+release their decryption key share for successful reveal, enables an environment where _censorship_,
+by not releasing decryption keys, and _collusion_, by peeking into encrypted messages before the
+reveal condition was reached, become very unlikely.
+
+In Shutter-Optimism this allows to build
```

Comment:
This sentence is somewhat odd to me. I would try to highlight that the underlying chain allows for MEV-resistant usage when transactions are shutterized. 

It should not be confused that the contracts are inherently MEV resistant or that they would need specific adaptions to be MEV resistant.